src/dscope.d

```diff
@@ -510,6 +510,25 @@ struct Scope
             return null;
         }
 
+        if (global.params.check10378)
+        {
+            // Search both ways
+
+            auto sold = searchScopes(flags | SearchCheck10378);
```

Comment:
OK, I noticed that we cannot get intentional result on this line.

When `-transition=checkimports` is specified, `global.params.bug10378` is set to false, so this `Dsymbol.search` will return the new lookup algorithm result always.